app/renderer/abr-document.js

```diff
@@ -87,6 +94,51 @@ function AbrDocument () {
             var fontSize = config.editor["font-size"] || "16px";
             that.setFontSize(fontSize);
 
+            // Events concerning AbrPane
+            that.cm.on("cursorActivity", function(cm) {
+              // Window title update
+              that.updateWindowTitle();
```

Comment:
Ok. I just copied the event handlers over from `abr-pane.js`. I can change this.